packages/web-app-files/src/mixins/actions/restore.js

```diff
@@ -26,6 +26,10 @@ export default {
             ) {
               return false
             }
+            if (!resources.every((r) => r.canBeRestored())) {
```

Comment:
Maybe we can use this logic more often. I think currently the user is being offered to perform certain batch actions that then fail somewhere down the selected items because permissions are missing 😅